docs/resources.rst

```diff
@@ -15,6 +15,12 @@ General
 - `Vyper greatest hits smart contract examples <https://github.com/pynchmeister/vyper-greatest-hits/tree/main/contracts>`_
 - `A curated list of Vyper resources, libraries, tools, and more <https://github.com/stars/pcaversaccio/lists/vyper>`_
 
+Editors
+-------
+
```

Comment:
maybe add the VScode plugin as well as the Vim plugins and such

Reply:
vyper-lsp is both VS Code and Vim, or is there some other plugin as well?

Reply:
https://github.com/vyperlang/vim-vyper

Reply:
yes to vim-vyper

for vscode vyper, tintinweb's version is slowly becoming unmaintained(?) but we could put in https://github.com/z80dev/vscode-vyper-lsp-extension